### PR TITLE
feat: add docs for Custom Domains for all platforms

### PR DESCRIPTION
**Pull request by david-mcafee**
_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/fragments/lib/graphqlapi/existing-resources.mdx
Comment (HuiSF) [inline src/fragments/lib/graphqlapi/existing-resources.mdx:22]: nit: could replace xxx with a bit more meaningful example. e.g.

```
https://api.yourdomain.com/graphql
```

**Review by HuiSF — APPROVED** [review]

File: src/fragments/lib/graphqlapi/existing-resources.mdx
Comment (david-mcafee) [reply src/fragments/lib/graphqlapi/existing-resources.mdx:22]: Sure, that works!

File: src/fragments/lib/graphqlapi/existing-resources.mdx
Comment (thisisabhash) [inline src/fragments/lib/graphqlapi/existing-resources.mdx:22]: Can we have a link to Appsync documentation for the format of the domains here, if possible?

Comment (david-mcafee) [reply src/fragments/lib/graphqlapi/existing-resources.mdx:22]: The custom domain can have any format, but has to end in `graphql`. I can reference that here.

Comment (david-mcafee) [reply src/fragments/lib/graphqlapi/existing-resources.mdx]: I'll also link to the AWS doc that will be published

**Review by sktimalsina — APPROVED** [review]

**Review by thisisabhash — APPROVED** [review]

**Review by sammartinez — CHANGES_REQUESTED** [review]
This looks good, lets wait to merge this until we get the go ahead from AppSync on this.

**Comment by david-mcafee** [comment]
> This looks good, lets wait to merge this until we get the go ahead from AppSync on this.

Thanks @sammartinez! Yep, I'll be waiting until we merge the changes for all categories, as well as the AWS docs.

**Review by sammartinez — APPROVED** [review]
Lets go live!!! LGTM 🌮

**Pull request merged**
